src/mcp/tools/auth/set_auth_claims.ts

```diff
@@ -0,0 +1,31 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { setCustomClaim } from "../../../gcp/auth.js";
+
+export const set_auth_claims = tool(
+  {
+    name: "set_auth_claims",
+    description: "Sets a list of specific claims for a user.",
+    inputSchema: z.object({
+      uid: z.string().describe("the UID or localId of the user to update"),
```

Comment:
What is a `localId`? I think it's always a uid, no?

Reply:
When the API returns with user info from identity platform, it always returns a `localId` rather than a UID

Reply:
Let's correct the output to always change "localId" to "uid" in MCP responses then. Nobody should have to know about that particular bit of sausage 🙃

Reply:
Fixed!